simple_salesforce/api.py

```diff
@@ -662,6 +691,8 @@ def _call_salesforce(self, method, url, **kwargs):
             'Authorization': 'Bearer ' + self.session_id,
             'X-PrettyPrint': '1'
         }
+        additional_headers = kwargs.pop('headers', dict())
+        headers.update(additional_headers or dict())
```

Comment:
You don't need `additional_headers` assignment. You can get away with doing it in a single line:

`headers.update(kwargs.pop('headers', dict()))`

The way you have it right now, if `headers` isn't a key in `kwargs`, the default value of `dict()` will be returned. Then in the `headers.update` call, the empty dict will evaluate to be Falsy, which will incur another needless `dict()` creation. Okay, so I'm being a little overly pedantic, but the one-liner is a little clearer to me.